src/NodaTime.TzdbCompiler/Tzdb/ParserHelper.cs

```diff
@@ -73,7 +73,7 @@ internal static long ConvertSecondsWithFractionalToTicks(string text)
                                          CultureInfo.InvariantCulture);
             if (number < 0.0 || number >= 60.0)
             {
-                throw new FormatException("seconds out of valid range of [0, 60): " + number);
+                throw new FormatException($"seconds out of valid range of [0, 60]: {number}");
```

Comment:
There was a small typo here.

Reply:
That's not a typo: `[a, b)` means a value _x_ such that _a_ ≤ _x_ < _b_. See e.g. https://en.wikipedia.org/wiki/Interval_(mathematics)#Terminology.

In this case, 0 is a valid number of seconds, but 60 isn't (but 59.9 would be).

Reply:
@malcolmr. Thanks for correction. I'm going to revert my bad change.